package.json

```diff
@@ -158,9 +158,5 @@
     "packages/node-runtime-worker-thread",
     "packages/browser-repl",
     "packages/connectivity-tests"
-  ],
-  "dependencies": {
-    "@cyclonedx/cyclonedx-library": "^6.6.0",
-    "npm-package-arg": "^11.0.2"
-  }
+  ]
```

Comment:
Sorry, left these in while cobbling together #1975 and forgot to take them out once I was done (and didn't end up needing them)